src/fjaelllada/db_sync.py

```diff
@@ -0,0 +1,141 @@
+    last_remot_change = client.info(remote_path)["modified"]
+    last_remote_time = datetime.strptime(last_remot_change, "%a, %d %b %Y %H:%M:%S %Z").timestamp()
+    print(f"{last_remote_time}")
+    print(f"{last_local_change}")
+    print(f"{last_sync}")
+    if last_sync < last_remote_time and last_sync < last_local_change:
+        return True
+    else:
+        return False
+
+
+def sync_files(remote_path, local_path, last_sync):
+    both_changed_since_last_sync = changed_since_last_sync(local_path, remote_path, last_sync)
+    if both_changed_since_last_sync:
+        sync_with_merge(remote_path, local_path)
+    else:
+        sync_without_backup(remote_path, local_path)
+
+def sync_without_backup(remote_path, local_path):
+    local_more_recent = client.is_local_more_recent(local_path, remote_path)
+    if local_more_recent:
+        upload_file(local_path, remote_path)
+    else:
+        download_file(remote_path, local_path)
+
+def sync_with_merge(remote_path, local_path):
+    remote_merge_file_path = "../../for_merge.txt"
+    download_file(remote_path, remote_merge_file_path)
+    merged_data = []
+    pattern = r"\s+"
+    with open(local_path, "r") as local_file, open(remote_merge_file_path, "r") as remote_merge_file:
+        for line_local, line_remote in zip(local_file, remote_merge_file):
+            if line_local == line_remote:
+                merged_data.append(line_local)
+            else:
+                if re.match(pattern, line_local) and not re.match(pattern, line_remote):
+                    merged_data.append(line_local)
+                elif not re.match(pattern, line_local) and re.match(pattern, line_remote):
+                    merged_data.append(line_remote)
+                else:
+                    merged_data.append(line_local)
+                    merged_data.append(line_remote)
+    with open(local_path, "w") as local_file:
+        local_file.writelines(merged_data)
+
+    upload_file(local_path, remote_path)
+
+def poll_changes():
+    last_sync = time.time()
+    get_sync_files_from_json()
+    old_local_changes = [os.path.getmtime(local_file) for local_file in local_file_paths]
+    old_remote_changes = [
+        datetime.strptime(client.info(remote_file)["modified"], "%a, %d %b %Y %H:%M:%S %Z").timestamp()
+        for remote_file in remote_file_paths
+    ]
+
+    while True:
+        local_changes = [os.path.getmtime(local_file) for local_file in local_file_paths]
+        remote_changes = [
+            datetime.strptime(client.info(remote_file)["modified"], "%a, %d %b %Y %H:%M:%S %Z").timestamp()
+            for remote_file in remote_file_paths
+        ]
+
+        for i, (local_file, remote_file) in enumerate(zip(local_file_paths, remote_file_paths)):
+            if old_local_changes[i] < local_changes[i] or old_remote_changes[i] < remote_changes[i]:
+                lock = FileLock("lock_" + local_file)
+                with lock:
```

Comment:
To match the other code (and this can be a single line):
```suggestion
                with FileLock(local_file + ".lock"):
```